### DISCUSSION TITLE
The mayBecome property is wrongly conceived

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1403] [Loans (LOAN) & Market Data (MD)]
The restriction on https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoanApplicationsTemporal/LoanApplicant that it must have a value for https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansTemporal/LoanApplicationsTemporal/mayBecome of type LoanBorrower means, that for correctness, a LoanBorrower individual must be created at the time of creating a LoanApplicant individual even though the application might be rejected. This mayBecome property is wrongly conceived since it's trying to represent a potential evolution that belongs at the class level and needs a structure not available in OWL.

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/pull/841_

**Comment by JurgenZiemer** [comment]
I agree with Pete and would even go further. We don’t need an object property mayBecome (domain LoanApplicant, range LoanBorrower). 

1. Indeed, the LoanBorrower would be instantiated together with the loan commitment https://spec.edmcouncil.org/fibo/ontology/LOAN/Loans/LoanBasicTerms/LoanBorrowerCommitment after the application is approved.
2. It would be redundant to **assert** an object property linking the borrower to the applicant PartyInRole. We can navigate from the loan to its application and vice versa, finding the parties.
3. Customizing ontologists can **infer** AgentInRole that may become borrowers, creating a defined class "ProspectiveBorrower" with LoanApplicant as sufficient condition. (there may be more criteria than LoanApplicant)

Robert asked me to comment on this issue. I didn’t study the latest dev version in-depth.

**Comment by trypuz** [comment]
@ElisaKendall, are you planning to take this issue into account while cleaning-up LOANs?

**Comment by ElisaKendall** [comment]
@trypuz - yes
